lib/checkPullRequestReview.js

```diff
@@ -381,9 +382,63 @@ const handleResponseToReview = async (context) => {
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ * @param {string[]} assignees
+ * @param {string} comment
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+
+//This is a function to check if there are more than 24 Hours with the Pr
+const isMaxReviewPeriodExceeded = async (review) => {
+  let timeOfReviewSubmission = review.submitted_at;
+  timeOfReviewSubmission = new Date(timeOfReviewSubmission);
+  let currentTime = new Date.now();
```

Comment:
I don't think `new Date.now()` works.

Reply:
Sorry for that It should be a new Date() only, as that automatically returns the current DateTime. Thanks for pointing this out